proposals/lip-0058.md

```diff
@@ -730,10 +728,7 @@ This function returns the slot number corresponding to the input timestamp.
 
 ```python
 def getSlotNumber(timestamp):
-    if timestamp < genesisTimestamp:
-        raise Exception('Input timestamp cannot be smaller than genesis block timestamp.')
-    elapsedTime = timestamp - genesisTimestamp
-    return floor(elapsedTime / BLOCK_TIME)
+    return floor(timestamp / BLOCK_TIME)
```

Comment:
Notation: should we use the `//` integer division (as done in other LIPs)?

Reply:
I changed it everywhere in LIP 0058 and LIP 0044